sql/src/main/java/io/crate/executor/transport/TransportShardAction.java

```diff
@@ -107,7 +106,8 @@ public Tuple call() throws Exception {
             //noinspection unchecked
             response = callable.call();
         } catch (Throwable e) {
-            throw Throwables.propagate(e);
+            assert e instanceof RuntimeException : "Expecting instance of RuntimeException here";
```

Comment:
wrapping any throwable into a runtime exception does not work here as all method usages does not honor wrapped exceptions (e.g. the retry logic)